benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh

```diff
@@ -15,9 +15,15 @@
 # limitations under the License.
 
 LOCUST="/usr/local/bin/locust"
-LOCUST_OPTS="-f /locust-tasks/tasks.py --host=$TARGET_HOST"
+LOCUST_OPTS="-f /locust-tasks/tasks.py "
 LOCUST_MODE=${LOCUST_MODE:-standalone}
 
+if [[ "$REQUEST_TYPE" = "grpc" ]]; then 
```

Comment:
from the below change, it only assumes JetStream as the only caller, since the tasks.py takes in the JetStream proto

Reply:
That's right. When we seek to support other gRPC model servers/move to a generic proto. Discussed further in a different comment.